modules/tinymce/src/plugins/image/test/ts/browser/plugin/MainTabTest.ts

```diff
@@ -47,4 +48,29 @@ describe('browser.tinymce.plugins.image.plugin.MainTabTest', () => {
       '</figure>'
     ));
   });
+
+  it('TINY-6400: render image_class_list as Dialog type "panel"', async () => {
+    const editor = hook.editor();
+    editor.settings.image_caption = false;
+    editor.execCommand('mceImage');
+
+    const dialog = await TinyUiActions.pWaitForDialog(editor);
+    const expected = {
+      ['.tox-form__group button[aria-label="Class"]']: 1,
+      ['.tox-form__grid--2col button[aria-label="Class"]']: 0
+    };
+    Assertions.assertPresence('Asserting presence', expected, dialog);
+  });
+
+  it('TINY-6400: render image_class_list and image_caption as Dialog type "group"', async () => {
+    const editor = hook.editor();
+    editor.execCommand('mceImage');
```

Comment:
Should we explicitly set `editor.settings.image_caption = true;`?

```suggestion
    const editor = hook.editor();
    editor.settings.image_caption = false;
    editor.execCommand('mceImage');
```

If so, we should probably set it back to `false` at the end of the test

Reply:
It's already set to true in the setup settings, so we shouldn't need to do that here. However, we do need to make sure the override is cleaned up in the previous test case.

Reply:
updated